Modules/_testcapimodule.c

```diff
@@ -2907,7 +2907,6 @@ test_macros(PyObject *self, PyObject *Py_UNUSED(args))
     static_assert(1 == 1, "bug");
     Py_BUILD_ASSERT(1 == 1);
```

Comment:
This was a test for `Py_BUILD_ASSERT`, now the macro is deprecated but still exists. I'm not sure if we should remove the test.

Reply:
Resolved by surrounding the test with push-disable-pop specific warning options.